lib/mixlib/shellout.rb

```diff
@@ -322,7 +363,8 @@ def parse_options(opts)
           self.log_tag = setting
         when 'environment', 'env'
           self.environment = setting || {}
-
+        when 'with_logon'
```

Comment:
Found the [origin](https://github.com/djberg96/win32-process/blob/ffi/lib/win32/process.rb#L404) for that name, I think